index.html

```diff
@@ -22,6 +22,12 @@ <h1> Search repos </h1>
             <button class="submit-button" type="submit"> Submit </button>
         </form>
         <div id="dinamicContent"> </div>
+        <ul class="repo-list">
+            <li class="skeleton-card m-3"> </li>
```

Comment:
Let's not use `m-3`. Instead, put a `gap` on the `repo-list`. This way, if you change it from there, it will modify in all places.

NOTE: also remove the `m-3` class when creating the cards in JS.